packages/modules/gcloud/src/spanner-emulator-container.test.ts

```diff
@@ -0,0 +1,57 @@
+import { getImage } from "../../../testcontainers/src/utils/test-helper";
+import { SpannerEmulatorContainer } from "./spanner-emulator-container";
+
+// select the fourth FROM in the Dockerfile (spanner emulator)
+const IMAGE = getImage(__dirname, 3);
+
+describe("SpannerEmulatorContainer", { timeout: 240_000 }, () => {
+  it("should start and expose endpoints", async () => {
+    // <example startup> {
+    const container = await new SpannerEmulatorContainer(IMAGE).withProjectId("test-project").start();
+    const grpcEndpoint = container.getEmulatorGrpcEndpoint();
+
+    expect(grpcEndpoint).toMatch(/localhost:\d+/);
+
+    await container.stop();
+    // }
+  });
+
+  it("should create and delete instance and database via helper", async () => {
```

Comment:
Do you mind moving the helper and its tests to separate files? I think it'd make the separation between the container and its utilities a bit more clear